modules/images-samples-operator-deprecated-image-stream.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/configuring-samples-operator.adoc
+
+
+[id="images-samples-operator-deprecated-image-stream_{context}"]
+= Removing deprecated image stream tags from the Cluster Samples Operator
+
+The Cluster Samples Operator leaves deprecated image stream tags in an image stream because users can have deployments that use the deprecated image stream tags.
+
+You can remove deprecated image stream tags by editing the image stream with the  `oc tag` command.
```

Comment:
s/oc tag/oc tag -d/g

The remove imagestream tag cmd like this:
$oc tag -d openshift/ruby:2.3
Deleted tag openshift/ruby:2.3